internal/controllers/build_sign_events_reconciler.go

```diff
@@ -0,0 +1,223 @@
+package controllers
+
+import (
+	"context"
+	"errors"
+	"fmt"
+
+	buildv1 "github.com/openshift/api/build/v1"
+	kmmv1beta1 "github.com/rh-ecosystem-edge/kernel-module-management/api/v1beta1"
+	ocpbuildbuild "github.com/rh-ecosystem-edge/kernel-module-management/internal/build/ocpbuild"
+	"github.com/rh-ecosystem-edge/kernel-module-management/internal/constants"
+	"github.com/rh-ecosystem-edge/kernel-module-management/internal/meta"
+	ocpbuildsign "github.com/rh-ecosystem-edge/kernel-module-management/internal/sign/ocpbuild"
+	"golang.org/x/exp/maps"
+	"golang.org/x/text/cases"
+	"golang.org/x/text/language"
+	v1 "k8s.io/api/core/v1"
+	k8serrors "k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/client-go/tools/record"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/builder"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
+	"sigs.k8s.io/controller-runtime/pkg/manager"
+	"sigs.k8s.io/controller-runtime/pkg/predicate"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+)
+
+const (
+	BuildSignEventsReconcilerName = "BuildSignEvents"
+
+	createdAnnotationKey = "kmm.node.kubernetes.io/created-event-sent"
+)
+
+type jobEvent struct {
```

Comment:
I think all those JobEvent should be rename with buildSignEvents. WDYT?

Reply:
I would leave it like this - IMO no need to make it less generic.

Reply:
But we are not talking about job event, we are talking about BuildSign event.  job is just an implementation for upstream, no?

Reply:
Build and Sign are both types of jobs. That is why the initial reconciler was called job - to accomodate any workload that the controller may create to produce images.
Renaming this type to `buildOrSignEvent` feels awkward imho.